bin/fedx-scripts.js

```diff
@@ -1,19 +1,65 @@
 #!/usr/bin/env node
+const presets = require('../lib/presets');
 
-process.env.PROJECT_ROOT = process.cwd();
```

Comment:
Removed this everywhere it's used and swapped for process.cwd() since we now have less trouble controlling which directory scripts are being executed in.